lib/init/features/functions/typescript.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(setup, config) {
       name: "lint",
       type: "confirm",
       message: "Do you want to use TSLint to catch probable bugs and enforce style?",
-      default: true,
+      default: false,
```

Comment:
This should be true. If they use TypeScript they'll have a precompile step even if they don't lint, so we may as well leave it true.